src/functions.js

```diff
@@ -17,6 +17,6 @@ export function COALESCE(...args) {
  * Return the size on an array as text, with an optional given prefix
  */
 export function BADGE(array, prefix = '') {
-  const length = array?.length || 0;
+  const length = array?.length || '';
   return `${prefix}${length}`;
```

Comment:
```suggestion
  const length = array?.length || '';
  if length:
     return `${prefix}${length}`;
  return '';
```

make it easy to return nothing.

Reply:
added postfix modified logic in in [ab2c955](https://github.com/huvrdata/evaluate/pull/20/commits/ab2c955160ee488e283e196d9063eee72302f48b)

This does modify the behavior tho: in my previous code it would still show the pre/postfix with an empty value, whereas now it doesn't. Seems fine either way but probably more correct to display nothing